gradle.properties

```diff
@@ -0,0 +1,22 @@
+version=4.0.4
+#
+# main dependencies
+#
+hivemq-extension-sdk.version=4.0.0
+simpleclient-dropwizard.version=0.9.0
+simpleclient-servlet.version = 0.9.0
+simpleclient-common.version= 0.9.0
+jetty-server.version= 10.0.0.beta2
+jetty-servlet.version= 10.0.0.beta2
+owner.version=1.0.10
+mockito-all.version=1.10.19
+junit.version=4.13
+commons-io.version=2.4
+hivemq-testcontainer.version=1.2.0
```

Comment:
please use the same order as you defined the dependencies also for the versions

Reply:
done